app/src/main/java/io/appium/uiautomator2/model/BaseElement.java

```diff
@@ -269,4 +270,25 @@ public static String getExtrasAsString(AccessibilityNodeInfo nodeInfo) {
         }
         return TextUtils.join(EXTRAS_SEPARATOR, extras);
     }
+
+    @Nullable
+    public static String getA11yActionsAsString(AccessibilityNodeInfo nodeInfo) {
+        StringBuilder actionsBuilder = new StringBuilder();
+        List<AccessibilityAction> actionList = nodeInfo.getActionList();
+        if (actionList.isEmpty()) {
+            return null;
+        }
+        for (AccessibilityAction action : actionList) {
+            String[] split = action.toString().split(" ");
```

Comment:
it would be useful for readability to add a comment with an example about how a raw value looks like before we split it